end_to_end/2-lineage-train-assess-bias-tune-registry-e2e.ipynb

```diff
@@ -0,0 +1,1083 @@
+{
```

Comment:
<p>Why do you look at only the specific feature <code>customer_gender_female</code>  class imbalance?</p><p>I think you should also mention about this notebook: https://github.com/aws/amazon-sagemaker-examples/blob/master/sagemaker_processing/fairness_and_explainability/fairness_and_explainability.ipynb</p><p> </p><p>This shows how to see the full bias report on Studio.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1974/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>

Reply:
<p>good idea. done.</p>

Reply:
<p>This is only an example of potential bias, we cannot delve into more detailed aspects of bias here other wise it will detract from our main goal of showing integration of services. we have taken one salient bias, class imbalance as the key example.</p>